it/router/AccountTypeForCredIdFeature.scala

```diff
@@ -48,7 +50,7 @@ class AccountTypeForCredIdFeature extends StubbedFeatureSpec with CommonStubs {
       verify(getRequestedFor(urlEqualTo("/auth/enrolments-uri")))
 
       And("user's details should not be fetched from User Details")
```

Comment:
all the comments are now wrong `user's details should be fetched from User Details` - remove the `not`